pom.xml

```diff
@@ -248,6 +248,18 @@
       <artifactId>jenkins-test-harness</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>git</artifactId>
+      <version>4.11.3</version>
+      <scope>compile</scope>
+    </dependency>
+      <dependency>
+          <groupId>org.jenkins-ci.plugins</groupId>
+          <artifactId>git</artifactId>
```

Comment:
Why is this here...twice?!

Reply:
Good question.  The comment on the commit indicates that it was intended to be temporary.  I've removed it, merged the master branch, and pushed the change